src/Compilation/Config.zig

```diff
@@ -101,7 +103,7 @@ pub const Options = struct {
     use_lib_llvm: ?bool = null,
     use_lld: ?bool = null,
     use_clang: ?bool = null,
-    lto: ?bool = null,
+    lto: LtoMode = .none,
```

Comment:
Sorry, I must've missed this earlier. Unlike the `lto` field on `Compilation.Config`, I think this field should be `?LtoMode`. Some of the checks below are interested in whether it's left unset, and `.none` (`-fno-lto`) is indistinguishable from it being unset.